tests/routers/test_storage.py

```
@@ -256,3 +256,48 @@ def test_google_drive_stats_endpoint(self, authenticated_client, db_session, mon
+        )
+        assert response.status_code == 200
+
+        location_from_db = (
+            db_session.query(ColdStorageLocation)
+            .filter(ColdStorageLocation.id == location.id)
+            .first()
+        )
+        assert location_from_db is not None
+        cfg = location_from_db.get_backend_config()
+        assert cfg["folder_id"] == "updated-folder-id"
+        assert cfg["refresh_token"] == "refresh-token"
```

Ruff (0.15.10)

[error] Possible hardcoded password assigned to: "refresh_token"

(S105)